compiler/linter/Types/SIloc.ml

```diff
@@ -0,0 +1,5 @@
+include Set.Make (struct
+  type t = Iloc.t
+
+  let compare = Iloc.compare
+end)
```

Comment:
Can you define a module in Iloc instead ?

Reply:
Other possibility, since this datatype (`Iloc.t`) is specific to the RD analysis, is to put `module SIloc = Set.Make(Iloc)` in `RDAnalyser.ml`.